### PR TITLE
Fix a bug where we highlighted all concept spans regardless of their score.

### PR DESCRIPTION
**Pull request by nsthorat**
https://lilacai-nikhil-staging.hf.space/datasets#local/OpenOrca-10k&query=%7B%22searches%22%3A%5B%7B%22path%22%3A%5B%22response%22%5D%2C%22type%22%3A%22concept%22%2C%22concept_namespace%22%3A%22lilac%22%2C%22concept_name%22%3A%22source-code%22%2C%22embedding%22%3A%22gte-small%22%7D%5D%7D

### CODE REVIEW

File: web/blueprint/src/lib/components/datasetView/spanHighlight.ts [hunk 1]
```diff
@@ -111,6 +111,8 @@ export function getMonacoRenderSpans(
         if (isConceptSearch || isSemanticSearch) {
```
Comment (brilee) [inline web/blueprint/src/lib/components/datasetView/spanHighlight.ts:111]: the above `let isHighlighted = ...` declaration seems to be completely overwritten in all branches of this if-else tree. Should it just say `let isHighlighted = false`?

**Review by brilee — APPROVED** [review]

File: web/blueprint/src/lib/components/datasetView/ItemMediaTextContent.svelte [hunk 2]
```diff
@@ -213,7 +213,7 @@
         },
         {
           value: `<span>${renderSpan.text}</span>`,
-          supportHtml: true,
+          supportHtml: false,
```
Comment (dsmilkov) [inline web/blueprint/src/lib/components/datasetView/ItemMediaTextContent.svelte:216]: this change intentional?

File: web/blueprint/src/lib/components/datasetView/spanHighlight.ts [hunk 3]
```diff
@@ -111,6 +111,8 @@ export function getMonacoRenderSpans(
...
           if ((value as number) > 0.5) {
             isHighlighted = true;
+          } else {
+            isHighlighted = false;
           }
         } else {
           isHighlighted = true;
```
Comment (dsmilkov) [inline web/blueprint/src/lib/components/datasetView/spanHighlight.ts:118]: do we highlight everything outside concept and semantic ?

**Review by dsmilkov — APPROVED** [review]

File: web/blueprint/src/lib/components/datasetView/ItemMediaTextContent.svelte (hunk 2 above)
Comment (nsthorat) [reply web/blueprint/src/lib/components/datasetView/ItemMediaTextContent.svelte:216]: yes, intentional

File: web/blueprint/src/lib/components/datasetView/spanHighlight.ts (hunk 1 above)
Comment (nsthorat) [reply web/blueprint/src/lib/components/datasetView/spanHighlight.ts:106]: done

File: web/blueprint/src/lib/components/datasetView/spanHighlight.ts (hunk 3 above)
Comment (nsthorat) [reply web/blueprint/src/lib/components/datasetView/spanHighlight.ts:113]: yea

**Pull request merged**